gridpapers.dtx

```diff
@@ -770,9 +769,11 @@
 \GP@declarehexpat
 \GP@declaretripat
 \GP@declareisopat
-\GP@declarelightconepat
 \GP@declaredotpat
 
+\usetikzlibrary{patterns.meta}
+\GP@declarelightconepat
+
```

Comment:
Is this needed? Why not keep the `\usetikzlibrary{patterns.meta}` up above (like it was originally), and keep the order of pattern declaration the same?

Reply:
See comment above on where it was removed. The 'early' GP@declares use pgfdeclarepatternformonly; GP@declarelightconepat uses tikzdeclarepattern, which is part of patterns.meta. This is a kludgy workaround for the fact that old versions of patterns.meta break pgf's pattern declaration.